scripts/sanitycheck

```diff
@@ -1847,10 +1848,8 @@ class FilterBuilder(CMake):
         if self.platform.name == "unit_testing":
             return {}
 
-        _generated_dt_confg = "include/generated/generated_dts_board.conf"
-
         cmake_cache_path = os.path.join(self.build_dir, "CMakeCache.txt")
-        dt_config_path = os.path.join(self.build_dir, "zephyr", _generated_dt_confg)
+        dts_path = os.path.join(self.build_dir, "zephyr", self.platform.name + ".dts.pre.tmp")
```

Comment:
Could move this variable next to where it's used maybe, to make the connection clearer.